ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/files/3scale_remove_tenants.yml

```diff
@@ -0,0 +1,18 @@
+
+# Delete the Temant Projects on Remove
+- set_fact:
+    ocp4_workload_lpe_3scale_tenant_gw_project: "{{ ocp4_workload_lpe_3scale_tenant_user_name_base }}-gw"
+
+- name: Remove tenant namespaces
+  k8s:
+    state: absent
+    kind: Project
+    api_version: project.openshift.io/v1
+    definition:
+      metadata:
+        name: "{{ item }}"
+        annotations:
+          openshift.io/description: ""
+          openshift.io/display-name: ""
+  loop:
+      - "{{ ocp4_workload_lpe_3scale_tenant_gw_project }}"
```

Comment:
If it's just one list in the loop you can add it to the `loop` line. Also this is probably going to fail the indentation check the way it's written right now.

Reply:
Updated without a list